### DISCUSSION TITLE
1.10 Windows wheels?

### DISCUSSION THREAD

**Discussion by stretch4x4** [Toblerity/Fiona#1373] [Q&A] [answered]
Following from https://github.com/Toblerity/Fiona/discussions/1367 and the 1.10.a2 release https://github.com/Toblerity/Fiona/releases/tag/1.10a2, just wanted to know if there is any visibility on the progress of the windows packaging for 1.10?

Some of our windows builds have picked up the pre-release and are failing I believe due to the lack of a windows specific wheel.
(The build is installing geopandas which has the requirement of `fiona>=1.8`)

> CRITICAL:root:A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.

If this is going to take a while to fix, we will have to pin an older version of Fiona across quite a few builds.

Thanks

**Comment by sgillies** [comment, marked as answer]
@stretch4x4 you'll have to work around the lack of Windows wheels for 1.10a2. I'll have a solution for the liblzma port by 1.10b1, but that's a couple weeks out.

**Reply by sgillies** [reply]
BTW, Windows wheels for 1.0b1 were uploaded on April 16.
